Doc/examples/Q9BZS1.dat

```diff
@@ -0,0 +1,739 @@
+ID   FOXP3_HUMAN             Reviewed;         431 AA.
```

Comment:
It is a mistake from my part, but I've removed it from my branch weeks ago and it shouldn't be here.

I'll look into removing this file from the PR.